bin/oref0-online.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/bash
 echo; echo Starting oref0-online.
+# if we are connected to wifi but don't have an IP, try to get one
+if iwgetid -r wlan0 | egrep -q "[A-Z]|[a-z]|[0-9]?"; then
```

Comment:
I think the `?` is identical to `+` (or leaving it off completely) in this case, since we're just substring matching within the SSID.  That said, I think your regex is a bit cleaner, so we can use that.